### PR TITLE
Add flink-shaded-7.0 release

### PR DESCRIPTION
**Pull request by sunjincheng121**
Add flink-shaded-7.0 release

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -77,34 +77,34 @@ flink_releases:
           name: "Pre-bundled Hadoop 2.4.1"
           category: "Pre-bundled Hadoop"
           scala_dependent: false
-          id: 180-bundled-hadoop-241
-          url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop2-uber/2.4.1-1.8.0/flink-shaded-hadoop2-uber-2.4.1-1.8.0.jar
-          asc_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop2-uber/2.4.1-1.8.0/flink-shaded-hadoop2-uber-2.4.1-1.8.0.jar.asc
-          sha_url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop2-uber/2.4.1-1.8.0/flink-shaded-hadoop2-uber-2.4.1-1.8.0.jar.sha1
+          id: bundled-hadoop-241-70
+          url: https://repo.maven.apache.org/maven2/org/apache/flink/flink-shaded-hadoop2-uber/2.4.1-7.0/flink-shaded-hadoop2-uber-2.4.1-7.0.jar
```
Comment (zentol) [inline _config.yml]: `hadoop2`-> `hadoop-2`

**Comment by sunjincheng121** [comment]
Thanks for the review @zentol 
I have updated the PR accordingly.
Best,
Jincheng

File: _config.yml [hunk 2]
```diff
@@ -262,11 +262,11 @@ flink_releases:
 component_releases:
   -
       source_release:
-          name: "Apache Flink-shaded 6.0"
+          name: "Apache Flink-shaded 7.0"
           id: "s60-download-source"
```
Comment (hequn8128) [inline _config.yml]: Should we change s60 to s70?

Comment (zentol) [reply _config.yml]: yes

Comment (sunjincheng121) [reply _config.yml]: should correct it when merging.

**Comment by sunjincheng121** [comment]
Thanks for all of your review @zentol @hequn8128 
I have updated the PR! :)
Best,
Jincheng

**Comment by hequn8128** [comment]
@sunjincheng121 Thanks for your update. Looks good from my side. +1 to merge if @zentol also thinks it is ok.

Best, Hequn

**Pull request closed without merging**